src/main/java/ro/developmentfactory/thedrones/controller/DroneStatusController.java

```diff
@@ -0,0 +1,42 @@
+package ro.developmentfactory.thedrones.controller;
+
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import ro.developmentfactory.thedrones.entity.DroneStatus;
+import ro.developmentfactory.thedrones.service.DroneStatusService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/dronesStatus")
+public class DroneStatusController {
+
+    @Autowired
+    private DroneStatusService droneStatusService;
+
+    // Save
+    @PostMapping
+    public DroneStatus saveDroneStatus(@Valid @RequestBody DroneStatus droneStatus) {
+        return droneStatusService.saveDroneStatus(droneStatus);
+    }
+
+    // Get all
+    @GetMapping
+    public List<DroneStatus> fetchAllDroneStatus() {
+        return droneStatusService.fetchDroneStatusList();
+    }
+
+    // Update
+    @PutMapping("/{idDroneStatus}")
+    public DroneStatus updateDroneStatus(@RequestBody DroneStatus droneStatus, @PathVariable Long idDroneStatus) {
+        return droneStatusService.updateDroneStatus(droneStatus, idDroneStatus);
+    }
+
+    // Delete
+    @DeleteMapping("/{idDroneStatus}")
+    public String deleteDroneStatus(@PathVariable Long idDroneStatus) {
+        droneStatusService.deleteDroneStatusById(idDroneStatus);
+        return "DroneStatus deleted";
+    }
```

Comment:
according to the task, we should have 
_Get current status response should contain drone id, current position (ex: [2, 6]) and facing direction (ex: north)_
it is for 1 drone, not a list, and we don't need the other operations as they are not controlled by the API directly - the values will change while a drone will move or rotate

Reply:
done